libsolidity/formal/SMTPortfolio.cpp

```diff
@@ -41,9 +41,15 @@ SMTPortfolio::SMTPortfolio(ReadCallback::Callback const& _readCallback)
 	m_solvers.emplace_back(make_shared<smt::CVC4Interface>());
 #endif
 #if !defined (HAVE_Z3) && !defined (HAVE_CVC4)
-	m_solvers.emplace_back(make_shared<smt::SMTLib2Interface>(_readCallback)),
+	m_solvers.emplace_back(make_shared<smt::SMTLib2Interface>(_smtlib2Responses));
+#else
+	if (!_smtlib2Responses.empty())
+		m_errorReporter.warning(
+			"Query responses for smtlib2 were given in the auxiliary input, "
```

Comment:
Actually what is the official styling, smtlib2 or SMTLib2?

Reply:
For messages like this maybe it is better to go with SMT-LIB2